Mathlib/Algebra/Category/ModuleCat/Basic.lean

```diff
@@ -432,11 +432,40 @@ end Module
 
 section
 
-variable {S : Type u} [CommRing S]
+universe u₀
+
+namespace Algebra
+
+variable {S₀ : Type u₀} [CommSemiring S₀] {S : Type u} [Ring S] [Algebra S₀ S]
 
 variable {M N : ModuleCat.{v} S}
 
-instance : Linear S (ModuleCat.{v} S) where
+/--
+Let `S` be an `S₀`-algebra. Then `S`-modules are modules over `S₀`.
+-/
+scoped instance : Module S₀ M := Module.compHom _ (algebraMap S₀ S)
+
+scoped instance : SMulCommClass S S₀ M where
+  smul_comm s s₀ n :=
+    show s • algebraMap S₀ S s₀ • n = algebraMap S₀ S s₀ • s • n by
+    rw [← smul_assoc, smul_eq_mul, ← Algebra.commutes, mul_smul]
```

Comment:
You should give `IsScalarTower` instead. And use `IsScalarTower.of_algebraMap_smul` in the proof.

Reply:
It still gives a large-import label. Perhaps add both `IsScalarTower` and `SMulCommClass` by hand?